keras_nlp/models/roberta/roberta_preprocessing_test.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for RoBERTa preprocessing layers."""
+
+import os
+
+import tensorflow as tf
+from absl.testing import parameterized
+from tensorflow import keras
+
+from keras_nlp.models.roberta.roberta_preprocessing import RobertaPreprocessor
+
+VOCAB_PATH = keras.utils.get_file(
```

Comment:
Can we do this with a small constructed vocabulary? Otherwise, we will need to annotate this so it does not run on github ci by default. Seems like it shouldn't be too tricky to mark a really small constructed vocabulary for the sentences below.